src/views/Register.vue

```diff
@@ -31,26 +24,72 @@ export default {
         username: '',
         password: '',
       },
+      usernameTaken: false,
     };
   },
 
   methods: {
     register() {
+      if (this.userData.username === '' || this.userData.password === '') {
+        window.alert('Please enter your username and password.');
```

Comment:
Kan je dan hier en bij die login ook in page errors van maken inplaats van alerts?